canonical-web-and-design/ubuntu.com
# Move device download pages to /download

## Done

- Move /download/iot/*pages* up one level 
- Update the navigation 
- Add redirects
- Move the copy docs
- Update the download dropdown

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/iot
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that all the individual download pages are not living at the /download root, for example `http://0.0.0.0:8001/download/iot/raspberry-pi` is now at `http://0.0.0.0:8001/download/raspberry-pi`

## Issue / Card

Fixes #6148
